package.json

```diff
@@ -19,12 +27,10 @@
     "cookbook": "cd polyfill && npm install && npm run cookbook && cd ..",
     "test262": "cd polyfill && npm install && npm run test262",
     "lint": "eslint . --ext js,mjs",
-    "build:prepare": "mkdirp out && mkdirp out/docs",
-    "build:polyfill": "cd polyfill && npm install && npm run build && cd ..",
-    "build:javascript": "npm run build:polyfill && cd docs && npm install && npm run build:javascript && cd .. && cp docs/*.js docs/*.js.map out/docs/",
-    "build:docs": "cd docs && npm install && npm run build:html && cd .. && cp docs/*.html docs/*.css out/docs/",
+    "build:javascript": "cd polyfill && npm install && npm run build && cd .. && rollup -m -c rollup.config.ejs",
+    "build:docs": "node build-docs.js",
     "build:spec": "ecmarkup spec.html out/index.html",
-    "prebuild": "mkdirp out && mkdirp out/docs",
+    "prebuild": "mkdir -p out/docs || mkdir out\\docs || rem 0",
```

Comment:
having to use the `\\` here makes the script imo less readable, and it makes it *much* more likely that someone will break it in the future by converting it back to `/`. Using the `mkdirp` package allows `/` to be used, which is the preferred/common path separator, and avoids this problem.

Reply:
so far 2 ppl want mkdirp dependency removed, and 1 wants it to stay.  can anyone else weigh in?

Reply:
who actively wants it removed (as opposed to merely being comfortable removing it)?

Reply:
i'm an imperfect human who created this pull-request partly to spite rejection of #437.  applying this particular change goes against that political-motivation.  so i'm asking if you can let this slide in light of that?  i can compromise on pretty much everything else asked.

Reply:
Why would that motivation be valuable? Not only is this a temporary polyfill until the proposal advances, but avoiding dependencies isn’t some kind of objectively good goal.

Separately, this is a dev dep - nobody sees it but the proposal champions who develop this package. Who is being helped by making npm install a few ms faster?